css/style.css

```diff
@@ -109,14 +110,16 @@ body {
 .network img {
   cursor: pointer;
 }
```

Comment:
- [ ] Please, make sure that the icon size and color is similar to those of the Figma Design.

Figma Design | Your Design
-------------|-------------
![image](https://user-images.githubusercontent.com/60979926/170319909-aae8e229-4041-4b19-ba41-5e4a67cdf609.png) | ![image](https://user-images.githubusercontent.com/60979926/170319328-73606b24-fd72-4daa-b9d3-7a2c1d98cd59.png)